packages/runtime-common/search-index.ts

```diff
@@ -528,32 +480,43 @@ export class SearchIndex {
   }
 
   private async loadFieldCard(
-    ref: ExportedCardRef,
+    ref: CardRef,
     fieldPath: string
   ): Promise<typeof Card> {
-    let api = await this.loadAPI();
-    let module: Record<string, typeof Card>;
+    let card: typeof Card | undefined;
     try {
-      module = await this.loader.import<Record<string, typeof Card>>(
-        ref.module
-      );
+      card = await loadCard(ref, { loader: this.loader });
     } catch (err: any) {
+      if (!("type" in ref)) {
```

Comment:
i don't understand this logic--wouldn't it be perfectly valid to have a field card that is an exported card ref? why are exported card refs as field invalid?

Reply:
or wait--never mind--this is inside a "catch" I missed that. this logic is sole to get a nice field name in the error message.